doc/api/zlib.md

```diff
@@ -712,6 +712,67 @@ The `zlib.bytesWritten` property specifies the number of bytes written to
 the engine, before the bytes are processed (compressed or decompressed,
 as appropriate for the derived class).
 
+### `zlib.crc32(data[, value])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `data` {string|Buffer|TypedArray|DataView} When `data` is a string,
+  it will be encoded as UTF-8 before being used for computation.
+* `value` {integer} An optional starting value. It must be a 32-bit unsigned
+  integer. **Default:** `0`
+* Returns: {integer} A 32-bit unsigned integer containing the checksum.
+
+Computes a 32-bit [Cyclic Redundancy Check][] checksum of `data`. If
+`value` is specified, it is used as the starting value of the checksum,
+otherwise, 0 is used as the starting value.
+
+The CRC algorithm is designed to compute checksums and to detect error
+in data transmission. It's not suitable for cryptographic authentication.
+
+To be consistent with other APIs, if the `data` is a string, it will
+be encoded with UTF-8 before being used for computation. If users only
+use Node.js to compute and match the checksums, this works well with
+other APIs that uses the UTF-8 encoding by default.
+
+Some third-party JavaScript libraries compute the checksum on a
```

Comment:
We might want to mention this is not a cryptographic hash, seems obvious but then again that's also true for md5/sha1 and people still use those.

Reply:
The documentation above already says "It's not suitable for cryptographic authentication."

Reply:
Ah, I missed that. That addresses my concern.